FormalConjectures/ErdosProblems/298.lean

```diff
@@ -33,3 +34,11 @@ https://github.com/b-mehta/unit-fractions -/
 theorem erdos_298 : (∀ (A : Set ℕ), 0 ∉ A → A.HasPosDensity →
     ∃ (S : Finset ℕ), S.toSet ⊆ A ∧ ∑ n ∈ S, (1 / n : ℚ) = 1) ↔ answer(True) := by
   sorry
+
+/--
+In [Bl21] it is proved under the weaker assumption that `A` onlu has positive upper density.
```

Comment:
```suggestion
In [Bl21] it is proved under the weaker assumption that `A` only has positive upper density.
```

Reply:
Note also that the version we formalised was with positive upper density (and so upper density is in that repo)

Reply:
Yes the definition `partialDensity` is inspired from there (if you didn't see already)!